ert_gui/tools/plot/plot_api.py

```diff
@@ -224,4 +219,18 @@ def history_data(self, key, case=None):
             history_key = f"{head}H:{tail}"
         else:
             history_key = f"{key}H"
-        return self.data_for_key(case, history_key)
+
+        df = self.data_for_key(case, history_key)
+
+        if not df.empty:
+            df = df.T
+            # Drop columns with equal data
+            duplicate_cols = [
+                cc[1] for cc in combi(df.columns, r=2) if (df[cc[0]] == df[cc[1]]).all()
+            ]
+            duplicate_cols = [
```

Comment:
isn't this a reassignment of the 228 duplicate_cols..? I struggle a bit to understand how this should work :thinking:

Reply:
fixed